src/lib/consts/routes.ts

```diff
@@ -0,0 +1,13 @@
+import { Routes } from '@lib/types'
+
+export const routes: Routes = {
+  home: '/',
+  signUp: 'rejestracja',
+  signIn: 'logowanie',
```

Comment:
`/` is missing here

Reply:
I think we should refactor routes, because some of them are in Polish and the rest are in English